### PR TITLE
{AKS} fix supportPlan calc in cli-preview

### PR DESCRIPTION
**Pull request by haitch**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

Sync the fix from official azure-cli, see PR [#27554](https://github.com/Azure/azure-cli/pull/27554).

### Related command
- `az aks update`

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

File: src/aks-preview/azext_aks_preview/tests/latest/test_managed_cluster_decorator.py [hunk 1]
```diff
@@ -6900,5 +6900,115 @@ def test_update_mc_profile_preview(self):
         dec_1.context.raw_param.print_usage_statistics()
 
 
+    def test_setup_supportPlan(self):
+        # default value in `aks_create`
+        ltsDecorator = AKSManagedClusterCreateDecorator(
```
Comment (FumingZhang) [inline src/aks-preview/azext_aks_preview/tests/latest/test_managed_cluster_decorator.py:6905]: The class is named as `AKSPreviewManagedClusterCreateDecorator` in aks-preview

**Review by FumingZhang — APPROVED** [review]
LGTM

please update `VERSION` in `setup.py` if you want to release a new version and leave some notes in `history.rst`.

File: src/aks-preview/HISTORY.rst [hunk 2]
```diff
@@ -9,9 +9,10 @@ If there is no rush to release a new version, please just add a description of t
 
 To release a new version, please select a new version number (usually plus 1 to last patch version, X.Y.Z -> Major.Minor.Patch, more details in `\doc <https://semver.org/>`_), and then add a new section named as the new version number in this file, the content should include the new modifications and everything from the *Pending* section. Finally, update the `VERSION` variable in `setup.py` with this new version number.
 
-Pending
+0.5.169
 +++++++
 * Vendor new SDK and bump API version to 2023-09-02-preview.
+* fix a bug in --support_plan handling when doing az aks update
```
Comment (zhoxing-ms) [inline src/aks-preview/HISTORY.rst:15]:
```suggestion
* fix a bug in --support-plan handling when doing az aks update
```

**Comment by zhoxing-ms** [comment]
Please take a look at this comment https://github.com/Azure/azure-cli-extensions/pull/6942#issuecomment-1791751665

**Comment by zhoxing-ms** [comment]
Please resolve this conflict~

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**